packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -346,6 +346,7 @@ export const RelationInputDataManager = ({
         'isSuccess'
       )}
       required={required}
+      toOneRelation={toOneRelation}
```

Comment:
What do you think about passing `canReorder` down instead? It makes the implementation a bit more independent ...